CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-<a name="3.0.4"></a>
+<a name="3.0.5"></a>
+# 3.0.5
+
+## OIDCStrategy
+
+### New features
+
+* added support of `id_token` in JWE compact serialization format. Supported key encryption algorithms (alg) are: 
```

Comment:
I would put some line breaks in here to make reading easier.
Supported key encryption algorithms (alg) are:
lkjlkjlkjk
Supported content encryption algorithms (enc) are:
lkjlkjlkjlkj